google/cloud/documentai_toolbox/converters/config/converter_helpers.py

```diff
@@ -0,0 +1,485 @@
+        prefix (str):
+            Required. The prefix for the location of the output folder.
+        annotation_file_prefix (str):
+            Required. The prefix to search for annotation file.
+        config_file_prefix (str):
+            Required. The prefix to search for config file
+
+    Returns:
+        List[Byte,Byte,Byte,str].
+
+    """
+
+    storage_client = document._get_storage_client()
+    bucket = storage_client.bucket(bucket_name=bucket_name)
+    blobs = storage_client.list_blobs(bucket_or_name=bucket_name, prefix=prefix)
+
+    metadata_blob = None
+
+    try:
+        for blob in blobs:
+            if "DS_Store" in blob.name:
+                continue
+            if not blob.name.endswith("/"):
+                blob_name = blob.name
+                file_name = blob_name.split("/")[-1]
+                if annotation_file_prefix in file_name:
+                    annotation_blob = blob
+                elif config_file_prefix in file_name:
+                    metadata_blob = blob
+                elif "pdf" in file_name:
+                    doc_blob = blob
+
+        if metadata_blob == None and config_path != None:
+            metadata_blob = bucket.get_blob(config_path)
+
+        print("Downloaded : %s\r" % prefix.split("/")[-1], end="")
+        return [
+            annotation_blob.download_as_bytes(),
+            doc_blob.download_as_bytes(),
+            metadata_blob.download_as_bytes(),
+            prefix.split("/")[-1],
+            file_name.split(".")[0],
+        ]
+    except Exception as e:
+        raise e
+
+
+def _upload_file(
+    bucket_name,
+    output_prefix,
+    file,
+) -> None:
+    r"""Uploads the converted docproto to gcs.
+
+    Args:
+        bucket_name (str):
+            Required. The bucket name.
+        output_prefix (str):
+            Required. The prefix for the location of the output folder.
+        file (str):
+            Required. The docproto file in string format.
+
+    Returns:
+        None.
+
+    """
+    storage_client = document._get_storage_client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(output_prefix)
+
+    print("Uploaded : %s\r" % output_prefix.split("/")[-1], end="")
+    blob.upload_from_string(file, content_type="application/json")
+
+
+def _get_files(blob_list, output_prefix, output_bucket, config_path: str = None):
+    download_pool = futures.ThreadPoolExecutor(10)
+    downloads = []
+    files = []
+    did_not_convert = []
+    labels = []
+    skip = ["DS_Store"]
+    prev = None
+    print("-------- Downloading Started --------")
+    for i, blob in enumerate(blob_list):
+        if "DS_Store" in blob.name:
+            continue
+        else:
+            file_path = blob.name.split("/")
+            file_path.pop()
+            doc_directory = file_path[-1]
+            file_path2 = "/".join(file_path)
+            if prev == doc_directory or f"{file_path2}/" == output_prefix:
+                continue
```

Comment:
I'd recommend using the os.path library instead of string manipulation

https://docs.python.org/3/library/os.path.html